.github/workflows/_e2e.yaml

```
@@ -23,7 +23,8 @@ jobs:
     timeout-minutes: 60
     # can't use env vars here
     runs-on: ${{ inputs.runner }}
-    if: ${{ inputs.runner != '' }}
+    # workaround for this bug https://github.com/actions/runner-images/issues/11985 - revert when done
+    if: ${{ fromJSON(inputs.matrix).k8sVersion != 'kindipv6' && inputs.runner != '' }}
```

Do we want to skip either on kindIpv6 or empty runner?
```suggestion
    if: ${{ fromJSON(inputs.matrix).k8sVersion != 'kindIpv6' || inputs.runner != '' }}
```

---

I read this as: We want to "run" when k8s version is not ipv6 and runner is not empty. I don't think your version is correct. We would run when `inputs.runner` is not empty so also when `kindIpv6`.

---

Why do we see 4 skips instead of 2 - https://github.com/kumahq/kuma/actions/runs/14444044925/job/40501259796?pr=13392 🤔